pkg/cmd/cli/cmd/version.go

```diff
@@ -126,13 +126,14 @@ func (o VersionOptions) RunVersion() error {
 		}
 
 		ocVersionBody, err := oClient.Get().AbsPath("/version/openshift").Do().Raw()
-		if err != nil && !(kapierrors.IsNotFound(err) || kapierrors.IsUnauthorized(err)) {
+		if err != nil && !(kapierrors.IsNotFound(err) || kapierrors.IsUnauthorized(err) || kapierrors.IsForbidden(err)) {
```

Comment:
When I was standing over your shoulder I thought it was surely a 404 since it was the older server ... maybe I'm crazy.